### ISSUE TITLE
Datatable: (scrollable): Wrong column size when used with Primefaces Extension component pe:layout

### ISSUE THREAD

**Issue by jaindsonvs** [primefaces/primefaces#5371]
Datatable: (scrollable): Wrong column size when used with Primefaces Extension component pe:layout

## 1) Environment
- PrimeFaces version: 7.0
- Does it work on the newest released PrimeFaces version? Version? No. It does not work on version 7.0 and primefaces-extensions 7.0.3. It used to work on Primefaces 6.2 and primefaces-extensions 6.2.4.
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) It does not work.
- Application server + version: the one used in https://github.com/primefaces/primefaces-test
- Affected browsers: Any.

## 2) Expected behavior

When no width given, all columns should have same size and be distributed equally over the available width and no scroll should be shown.

## 3) Actual behavior

All columns have same size, but wrong width, then we see a scroll. It is like the pe:layoutPane on west side is ignored when the size of the columns is calculated.

## 4) Steps to reproduce

Run "mvn clean jetty:run-exploded" on sample. Go to page http://localhost:8080/primefaces-test/test.xhtml and see the scroll in dataTable. If you change the dependencies on pom.xml to Primefaces version 6.2 and primefaces-extensions 6.2.4 it works as expected.

## 5) Sample XHTML

Download demo [here](https://docs.google.com/uc?id=1larkK5HXceb_xr3kDWBZ5TRgFCFSXXDi&export=download).

## 6) Sample bean

Download demo [here](https://docs.google.com/uc?id=1larkK5HXceb_xr3kDWBZ5TRgFCFSXXDi&export=download).

**Comment by melloware** [comment]
Confirmed. In 7.0 it generates a Table Header TH for the headers with a calculated width.
```html
<th id="j_idt9:j_idt10_clone" class="ui-state-default"  scope="col" style="width: 167px;">
<span class="ui-column-title">Column 1</span>
</th>
```
in 6.2 it does not it only generates TR with not TH.  So most likely something that was fixed in 7.0 related to datatable headers.  
```html
<tr data-ri="9" class="ui-widget-content ui-datatable-odd" role="row">
```
Most likely one of these issues: https://github.com/primefaces/primefaces/issues?utf8=%E2%9C%93&q=datatable+milestone%3A7.0+

**Comment by melloware** [comment]
@jaindsonvs I believe what you are seeing is the new correct behavior based on all the bugfixes for other datatable issues.   I think you will either have to give your column sizes or not use a scrollable table if you don't want those defaults.
